CONTRIBUTING.md

```diff
@@ -33,9 +33,9 @@ Once you've confirmed that the bug still exists in the latest version, you'll wa
 
 If you've upgraded to the latest version and you can't find it in our open issues list, then you'll need to tell us how to reproduce it Provide as much information as you can. You may think that the problem lies with your query, when actually it depends on how your data is indexed. The easier it is for us to recreate your problem, the faster it is likely to be fixed.
 
-### Feature Requests
+### Feature Requests & Proposals
 
-If you've thought of a way that OpenSearch could be better, we want to hear about it. We track feature requests using GitHub, so please feel free to open an issue which describes the feature you would like to see, why you need it, and how it should work.
+If you've thought of a way that OpenSearch could be better, we want to hear about it. We track `feature requests` using GitHub, so please feel free to open an issue which describes the feature you would like to see, why you need it, and how it should work. If you would like contribute code toward building it, you might consider a `feature proposal` instead. A feature proposal is the first step to helping the community better understand what you are planning to contribute, why it should be built, and collaborate on ensuring you have all the data points you need for implementation.
 
```

Comment:
I'm big on examples to provide nuance, what would you think about changing the tick'ed terms `feature requests` and `feature proposal` into links like this one for [feature proposals](https://github.com/search?q=org%3Aopensearch-project+%22How+did+you+come+up+with+this+proposal%3F%22&type=Issues) that have examples on the other sides of them?

Reply:
That's a GREAT idea! I'll update shortly

Reply:
Updated. I wanted to be explicit that the links would be to examples of feature requests and feature proposals, rather than linking the ticked items, which could be either examples, proposal templates, or create